Application/learn_static.py

```
@@ -64,42 +65,40 @@ def __init__(self, model, classes, X_test, Y_test, epochs, filename, prefix, out
-            result[i, 2] = np.median(Y.sum(axis=1)) #distance.mean()
+            result[i, 2] = np.median(Y.sum(axis=1))
             result[i, 3] = (np.argmax(Y, axis=1) == i).sum() / len(Y)
-        return result, predictions
+        
+        return result
     
-    def update_status(self, print_out = False, logs = {}):
-        description = "epoch "+str(min(self.epoch+1, self.epochs))+"/"+str(self.epochs)
-        if np.shape(self.X_test)[-1] > 0 and len(self.worst_values) > 0:
-            description += " -- worst acc/class: {0:.3f}".format(self.worst_values[-1])
+    def update_status(self, print_out = False, logs = {}, patience=5):
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)